src/main/js/store/reducers/globalConfiguration.js

```diff
@@ -125,7 +125,7 @@ const globalConfiguration = (state = initialState, action) => {
                 fetching: false,
                 updateStatus: 'UPDATED',
                 testing: false,
-                actionMessage: 'Update successful',
+                actionMessage: 'Save successful',
```

Comment:
The button is called "Save" and that covers both cases rather than mislabeling the POST case.